code/game/objects/items/powerfist.dm

```diff
@@ -37,6 +38,10 @@
 			if(IT.volume <= 3)
 				to_chat(user, "<span class='warning'>\The [IT] is too small for \the [src].</span>")
 				return
+			if(!lock)
+				ADD_TRAIT(src, TRAIT_NODROP, POWERFIST_STICK)
+				lock ++
```

Comment:
I'd prefer lock = LOCKED_POWERFIST or something along those lines.
Create a define and undefine in the file.